app/flatpak-builtins-uninstall.c

```diff
@@ -307,12 +307,19 @@ flatpak_builtin_uninstall (int argc, char **argv, GCancellable *cancellable, GEr
 
           if (ref_dir_pairs->len == 0)
             {
-              g_set_error (error, FLATPAK_ERROR, FLATPAK_ERROR_NOT_INSTALLED,
+              if (n_prefs == 1)
+              {
+                g_set_error (error, FLATPAK_ERROR, FLATPAK_ERROR_NOT_INSTALLED,
                            _("%s/%s/%s not installed"),
                            id ? id : "*unspecified*",
                            arch ? arch : "*unspecified*",
                            branch ? branch : "*unspecified*");
-              return FALSE;
+                return FALSE;
+              }
+              
+              g_printerr("Warning: %s is not installed\n", pref);
```

Comment:
This needs to be translated too, so add _() around the string. And also, there is a missing space before the first parenthesis